.appveyor.yml

```diff
@@ -1,8 +1,8 @@
 version: 0.2.7.{build}
 os: Windows Server 2012 R2
 install:
-- C:\Python34\Scripts\pip install ply pyreadline nose pygments prompt_toolkit
-- C:\Python35\Scripts\pip install ply pyreadline nose pygments prompt_toolkit
+- C:\Python34\Scripts\pip install pyreadline nose pygments prompt_toolkit
+- C:\Python35\Scripts\pip install pyreadline nose pygments prompt_toolkit
```

Comment:
We probably should still install ply so that we are testing the latest version on pypi